pennylane/hf/convert.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed for computing the molecular Hamiltonian.
+"""
+import numpy as np
+import openfermion
+import pennylane as qml
+from openfermion import QubitOperator
+from pennylane import Hamiltonian
+from pennylane.wires import Wires
+
+
+def _process_wires(wires, n_wires=None):
+    r"""Checks and consolidates custom user wire mapping into a consistent, direction-free, `Wires`
+    format. Used for converting between OpenFermion qubit numbering and Pennylane wire labels.
+
+    Since OpenFermion's qubit numbering is always consecutive int, simple iterable types such as
+    list, tuple, or Wires can be used to specify the 2-way `qubit index` <-> `wire label` mapping
+    with indices representing qubits. Dict can also be used as a mapping, but does not provide any
+    advantage over lists other than the ability to do partial mapping/permutation in the
+    `qubit index` -> `wire label` direction.
+
+    It is recommended to pass Wires/list/tuple `wires` since it's direction-free, i.e. the same
+    `wires` argument can be used to convert both ways between OpenFermion and Pennylane. Only use
+    dict for partial or unordered mapping.
+
+    Args:
+        wires (Wires, list, tuple, dict): User wire labels or mapping for Pennylane ansatz.
```

Comment:
Is wire mapping property of ansatz or the device/observable? 🤔

Reply:
Yes, it is a device property. Modified the text.